### ISSUE TITLE
Does not work with the new Pantheon CLI

### ISSUE THREAD

**Issue by ibuildit** [kalamuna/terminatur#32]
Terminus is deprecated, Pantheon CLI is the new kid in town, as a result, terminatur does not work, can not connect to Pantheon:

Dashboard unavailable: couldn't connect to host                      [error]
No user UUID found. Please authenticate first using 'drush pauth'    [error]
curl_setopt(): 77 is not a valid cURL handle resource                [warning]
terminus.drush.inc:3540
curl_setopt(): 77 is not a valid cURL handle resource                [warning]
terminus.drush.inc:3541
curl_setopt(): 77 is not a valid cURL handle resource                [warning]
terminus.drush.inc:3542
curl_setopt_array(): 77 is not a valid cURL handle resource          [warning]
terminus.drush.inc:3561
curl_exec(): 77 is not a valid cURL handle resource                  [warning]
terminus.drush.inc:3567
curl_errno(): 77 is not a valid cURL handle resource                 [warning]
terminus.drush.inc:3574
curl_getinfo(): 77 is not a valid cURL handle resource               [warning]
terminus.drush.inc:3580
No user UUID found. Please authenticate first using 'drush pauth'    [error]

**Comment by ibuildit** [comment]
It is also not possible to crush a local site as that command is dependent on authentication.

**Comment by pirog** [comment]
Good call. We've noted that on the kalastack project readme but we should
note that here as well.

On Saturday, March 7, 2015, ibuildit <notifications@github.com> wrote:

> Terminus is deprecated, Pantheon CLI is the new kid in town, as a result,
> terminatur does not work, can not connect to Pantheon:
>
> Dashboard unavailable: couldn't connect to host [error]
> No user UUID found. Please authenticate first using 'drush pauth' [error]
> curl_setopt(): 77 is not a valid cURL handle resource [warning]
> terminus.drush.inc:3540
> curl_setopt(): 77 is not a valid cURL handle resource [warning]
> terminus.drush.inc:3541
> curl_setopt(): 77 is not a valid cURL handle resource [warning]
> terminus.drush.inc:3542
> curl_setopt_array(): 77 is not a valid cURL handle resource [warning]
> terminus.drush.inc:3561
> curl_exec(): 77 is not a valid cURL handle resource [warning]
> terminus.drush.inc:3567
> curl_errno(): 77 is not a valid cURL handle resource [warning]
> terminus.drush.inc:3574
> curl_getinfo(): 77 is not a valid cURL handle resource [warning]
> terminus.drush.inc:3580
> No user UUID found. Please authenticate first using 'drush pauth' [error]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalamuna/terminatur/issues/32>.
>


-- 
Cheers,

Mike Pirog
Kalamuna
www.kalamuna.com

**Comment by ibuildit** [comment]
Do you think this will be fixed? Terminatur is really really handy...

**Comment by pirog** [comment]
All of our effort is currently going into the Kalabox2 project over at http://github.com/kalabox so its doubtful. 

That said, we plan on having terminatur-like functionality backed into KB2 soon. 

I've updated the README here to reflect the current project status
https://github.com/kalamuna/terminatur/commit/42319d9e74b86effd54c74e3d635ec84f0666cda

**Comment by ibuildit** [comment]
Sounds fantastic :)
